src/data_table/local_data_table.js

```diff
@@ -537,8 +542,11 @@ class LocalDataTable extends View {
   }
 
   _dataTableConfig() {
-    let displayStart = this._getSafeDisplayStartFromPageNumber();
-    let recordTotal = displayStart + this.paginateLength;
+    let displayStart, recordTotal;
+    if (this.urlPagination) {
+      displayStart = this._getSafeDisplayStartFromPageNumber();
+      recordTotal = displayStart + this.paginateLength;
+    }
```

Comment:
is it okay to pass `undefined` to iRecordsTotal and iRecordsDisplay below in the event urlPagination is off? (Versus omitting those params entirely). I assume so but wanted to check since before you were always sending 0 + length

Reply:
If we pass undefined, I was thinking that it doesn't get mapped on the data table side of things, hence it's back to being a normal data table. That might be a wrong interpretation however.